addon/serializers/application.js

```diff
@@ -86,7 +86,7 @@ export default DS.RESTSerializer.extend({
         // the links property so the adapter can async call the
         // relationship.
         // The adapter findHasMany has been overridden to make use of this.
-        if(options.relation) {
```

Comment:
I just meant `options.relationship` vs `options.array`. I recall now, they are two different options for how to persist a relationship. One with an array of ids, one with w URL.